src/include/migraphx/op/allocate.hpp

```diff
@@ -48,9 +48,9 @@ namespace op {
  */
```

Comment:
Suggested modification:

 - * Either `this.s` or `this.buf_type` must be set to calculate the dynamic output shape at compute
 - * time. 

 * For dynamic shapes, either `this.s` or `this.buf_type` must be set to calculate the dynamic output shape at compute
 * time, but not both.